GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -197,10 +200,29 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            EditorGUILayout.TextField("c:\\base.apk", GUILayout.MinWidth(FieldMinWidth));
+            Il2cppBuilder.ApkPathName =
+                EditorGUILayout.TextField(Il2cppBuilder.ApkPathName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                Il2cppBuilder.ApkPathName = EditorUtility.SaveFilePanel("Choose file name and location",
+                    Path.GetDirectoryName(Il2cppBuilder.ApkPathName), Path.GetFileName(Il2cppBuilder.ApkPathName),
+                    "apk");
+            }
+
             EditorGUILayout.EndHorizontal();
             EditorGUILayout.Space();
-            GUILayout.Button("Build Base APK", GUILayout.Width(ButtonWidth));
+
+            if (!Il2cppBuilder.ProjectIsUsingIl2cpp)
+            {
+                EditorGUILayout.LabelField(
+                    "Warning: You are not using IL2CPP as the project's scripting backend, and this may result into a larger apk size than necessary.",
```

Comment:
IL2CPP by itself is often bigger than Mono. To make it smaller you need IL2CPP + engine stripping.

Reply:
Resolved.